### PR TITLE
Movie form budget

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
Second Generation of Form Budget + move static-model.

### CODE REVIEW

File: libs/utils/src/lib/common-interfaces/range.ts [hunk 1]
```diff
@@ -16,6 +17,7 @@ export function createRange<D>(params: Partial<RawRange<D>> = {}): RawRange<D> {
   return {
     from: null,
     to: null,
+    label: '',
```
Comment (phpgeekfr) [inline libs/utils/src/lib/common-interfaces/range.ts]: need to initialize label even if not required (`label?`)?

File: libs/utils/src/lib/common-interfaces/range.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 export interface RawRange<D> {
   from: D | null;
   to: D | null;
+  label?: string;
```
Comment (phpgeekfr) [inline libs/utils/src/lib/common-interfaces/range.ts]: please tell me (or Clélia) when you change models.

Comment (GrandSchtroumpf) [reply libs/utils/src/lib/common-interfaces/range.ts]: Clélia changed that.

Comment (phpgeekfr) [reply libs/utils/src/lib/common-interfaces/range.ts]: OK. Draw.io was not up to date.

**Pull request merged**